source/index.html.md

```diff
@@ -200,6 +200,7 @@ error categories, the last two digits define specific errors.
 * 0366: Two factor authentication method is not enabled
 * 0367: Unable to generate quote at the present time
 * 0368: Rejected withdrawal to restricted address
+* 0369: Incorrect device id (non-existent or does not belong to the user)
```

Comment:
Not sure if it's good idea to use the same error code for different reasons. For final users I think it's better to have a fine grained error code/description.

Reply:
Not sure I'm following, error code `0359` is new, while the description is from my point of view very descriptive. Can you please expand on the idea?

Reply:
looks like the message is for either 'an incorrect device id' **OR** 'does not belong to the user'

Reply:
I'd leave it as just `incorrect device id` then

Reply:
@domix @thenewversion - thanks for your input. Changes addressed.